src/fmi_export/fmiCollector.cpp

```diff
@@ -52,15 +59,23 @@ change_code fmiCollector::trigger(coreTime time)
 
 void fmiCollector::set(const std::string &param, double val)
 {
-	if (param.front() != '#')
+	if (param.front() == '#')
```

Comment:
This is should really be `if (param.empty())`  if param is empty this will cause a seg fault

Reply:
there are probably other places this shows up that should be changed as well

Reply:
There are a lot of places where params are getting parsed. I have them down as something to look over, but that's going to be a little later. (This diff is just a checkout of the file from cmake_update)